plugins/inputs/zipkin/codec/jsonV1/jsonV1.go

```diff
@@ -220,7 +220,7 @@ func TraceIDFromString(s string) (string, error) {
 	var hi, lo uint64
 	var err error
 	if len(s) > 32 {
-		return "", fmt.Errorf("TraceID cannot be longer than 32 hex characters: %s", s)
+		return "", fmt.Errorf("length of TraceID cannot be greater than 16 hex characters: %s", s)
```

Comment:
Doesn't this change what was said before (16 vs 32 chars)?

Reply:
Yes, my bad.